<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSecilenleriAlHobiler">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçilenleri Al</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResultHobiler">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupHobiler">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>151</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbSinema">
        <property name="text">
         <string>Sinema</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKitap">
        <property name="text">
         <string>Kitap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSpor">
        <property name="text">
         <string>Spor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupDersler">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>151</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbWeb">
        <property name="text">
         <string>Web Tasarım</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbProgramlama">
        <property name="text">
         <string>Programlama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMatematik">
        <property name="text">
         <string>Matematik</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSecilenleriAlDersler">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçilenleri Al</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResultDersler">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
